tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -91,6 +93,8 @@ pub mod pallet {
 		IdentityNotExist,
 		/// creating the prime identity manually is disallowed
 		LinkPrimeIdentityDisallowed,
+		/// removing the prime identity manually is disallowed
+		RemovePrimeIdentityDisallowed,
```

Comment:
I don't think this one is used, plus we should actually allow removing the prime identity (which removes the whole IDGraph)

Reply:
good catch, thanks 🙌